api/client/service.proto

```diff
@@ -2,8 +2,10 @@ syntax = "proto3";
 
 package nighthawk.client;
```

Comment:
Before I review the changes here I have a few questions that can help me to understand this better. What is the definition of a "client" and a "server" when it comes to this RPC service. Do I understand it correctly that NH acts as the client and this proto file defines a server that should be implemented by the users who want to use NH in performance tests?

I.e. is it NH that will call the defined RPC methods? Would you mind adding a top level file comment with some background and few notes about the architecture so that this is clear?

Reply:
@mum4k Please see "Executables / nighthawk_service" over at https://docs.google.com/document/d/1k21xIVCxQX51OORw_WQYjRLzi0m7pM1MN0859XXqWxI/edit?usp=sharing

(note: doc is a work in progress)